test/testutil.lua

```diff
@@ -392,7 +392,7 @@ function M.check_logs()
   )
 end
 
-function M.sysname()
+local function sysname()
   return uv.os_uname().sysname:lower()
```

Comment:
not used in any existing tests, and they should not really be using this directly.